src/types/st-object.ts

```diff
@@ -1,11 +1,53 @@
 import { Field, FieldInstance } from "../enums";
 import { SerializedType, JsonObject } from "./serialized-type";
+import {
+  xAddressToClassicAddress,
+  isValidXAddress,
+} from "ripple-address-codec";
 import { BinaryParser } from "../serdes/binary-parser";
 import { BinarySerializer, BytesList } from "../serdes/binary-serializer";
 
-const OBJECT_END_MARKER = Buffer.from([0xe1]);
-const OBJECT_END_MARKER_NAME = "ObjectEndMarker";
-const OBJECT_FIELD_TYPE_NAME = "STObject";
+const OBJECT_END_MARKER_BYTES = Buffer.from([0xe1]);
```

Comment:
This looks like one byte, so why is the name `BYTES` (plural)?

```suggestion
const OBJECT_END_MARKER_BYTE = Buffer.from([0xe1]);
```

(If renamed, all uses of this constant would also need to be changed.)